test/NCI.OCPL.Api.Glossary.Tests/Tests/Controllers/TermsControllerTest.cs

```diff
@@ -126,72 +128,232 @@ public async void GetById()
         }
 
         [Fact]
-        public async void GetById_BlankRequiredFields()
+        public async void GetById_WithFallback_GeneticsHP()
         {
```

Comment:
As with the other tests for the fallback logic, coming in cold (or oh not looking at the code recently), you have to read through a lot of set up code in order to learn what this method is testing.  The same applies to the patient fallback, and there doesn't appear to be a test for the "NotSet" dictionary.